.github/workflows/app_build.yaml

```diff
@@ -0,0 +1,49 @@
+name: NoPorts Desktop Builds
+
+on:
+  push:
+    branches:
+      - trunk
+permissions: # added using https://github.com/step-security/secure-repo
+  contents: read
+
+jobs:
+  windows_build:
+    name: Windows Build
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+          flutter-version-file: "3.24.5"
+      - run: flutter pub get
+
+      - name: Build msix file with Dart SDK
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: stable
+      - run: dart run msix:create
```

Comment:
prefer calling all of the uses first, then do all of the runs.

Reply:
You also don't need to setup dart if you've already setup flutter. Dart binary is installed with Flutter.